FinSearchUnified/Tests/Components/ConfigLoaderTest.php

```diff
@@ -244,14 +244,12 @@ public function testCacheWithCorrectKey($cacheTestResponse, $loadCallCount)
     public function testCacheWithNonExistingKey($cacheResponse, $loadCallCount)
     {
         $httpClient = Shopware()->Container()->get('http_client');
```

Comment:
This change should not have been necessary. Please re-add it.

Reply:
It was necessary because the shopkey is changed mid-execution which can cause the expectation defined in this test to fail - since the cacheKey is dependent on the shopkey then it changes the generated key which can cause problems. 
This change does not change the outcome as it is already being tested.